docs/streams/developer-guide/config-streams.html

```diff
@@ -270,21 +296,26 @@ <h4><a class="toc-backref" href="#id5">bootstrap.servers</a><a class="headerlink
               <td colspan="2">The amount of time in milliseconds, before a request is retried. This applies if the <code class="docutils literal"><span class="pre">retries</span></code> parameter is configured to be greater than 0. </td>
               <td>100</td>
           </tr>
-          <tr class="row-even"><td>rocksdb.config.setter</td>
+          <tr class="row-odd"><td>rocksdb.config.setter</td>
             <td>Medium</td>
             <td colspan="2">The RocksDB configuration.</td>
             <td></td>
           </tr>
-          <tr class="row-odd"><td>state.cleanup.delay.ms</td>
+          <tr class="row-even"><td>state.cleanup.delay.ms</td>
             <td>Low</td>
             <td colspan="2">The amount of time in milliseconds to wait before deleting state when a partition has migrated.</td>
             <td>600000 milliseconds</td>
           </tr>
-          <tr class="row-even"><td>state.dir</td>
+          <tr class="row-odd"><td>state.dir</td>
             <td>High</td>
             <td colspan="2">Directory location for state stores.</td>
             <td><code class="docutils literal"><span class="pre">/tmp/kafka-streams</span></code></td>
           </tr>
+          <tr class="row-odd"><td>topology.optimization</td>
```

Comment:
Yep, thanks (to be honest I'm not sure what this even does, since a few of these were incorrect already and no one ever noticed AFAICT. But better to be correct)

Reply:
I guess it tells the renderer which CSS to use to have alternating row colours. Apparently, CSS3 already offers ways to accomplish this without these classes (see https://www.textfixer.com/tutorials/css-table-alternating-rows.php).